Code/GraphMol/TautomerQuery/TautomerQuery.h

```diff
@@ -74,7 +74,7 @@ class RDKIT_TAUTOMERQUERY_EXPORT TautomerQuery {
 
   // Query fingerprint
   ExplicitBitVect *patternFingerprintTemplate(unsigned int fpSize = 2048U);

```

Comment:
Do you need the non-`const` version of this method at all?
The two implementations look identical.

Reply:
might as well just remove the non-const one.

Reply:
We need the non-const one for serialization.